### PR TITLE
Update readme cn

### PR DESCRIPTION
**Pull request by SamitHuang**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -14,91 +14,137 @@
...
+    --opt eval.dataset_root={path/to/your_dataset} eval.ckpt_load_path={path/to/ckpt_file}
+```
 
-## 模型列表
+更多使用方法，请参考[教程](#教程)中的模型训练章节。
+
+## 教程
+
+- 数据集
+    - [数据集准备](tools/dataset_converters/README.md)
```
Comment (Songyuanwei) [inline README_CN.md:130]: 这里的链接如果有的话，是否需要链接中文readme_cn.md， 下同

File: README_CN.md [hunk 2]
```diff
@@ -107,54 +153,50 @@ MindOCR提供易用的文本检测识别推理工具，支持CPU/GPU/Ascend 910
...
+当前已在模型训练评估中验证过的公开OCR数据集如下。
 
 <details open>
-<summary>文本检测</summary>
-
-- [x] ICDAR2015 [论文]((https://rrc.cvc.uab.es/files/short_rrc_2015.pdf)) [主页]((https://rrc.cvc.uab.es/?ch=4)) [下载说明](docs/cn/datasets/icdar2015_CN.md)
+<summary>通用OCR数据集</summary>
 
-- [x] Total-Text [论文](https://arxiv.org/abs/1710.10400) [主页](https://github.com/cs-chan/Total-Text-Dataset/tree/master/Dataset) [下载说明](docs/cn/datasets/totaltext_CN.md)
+- [x] [ICDAR2015](https://rrc.cvc.uab.es/?ch=4) [[paper](https://rrc.cvc.uab.es/files/short_rrc_2015.pdf)] [[download](docs/en/datasets/icdar2015.md)]
```
Comment (Songyuanwei) [inline README_CN.md:181]: 这里是否需要改成中文？paper -> 论文， download ->下载说明。

File: README_CN.md [hunk 3]
```diff
@@ -14,91 +14,137 @@
...
 
-#### 2.1 使用MindSpore Lite和ACL推理(Ascend 310)
+可以看到图像中的文字块均被检测出来并正确识别。更详细的用法介绍，请参考推理[教程](#教程)。
 
-MindOCR集成了[MindSpore Lite](https://www.mindspore.cn/lite)和[ACL](https://www.hiascend.com/document/detail/zh/canncommercial/63RC1/inferapplicationdev/aclcppdevg/aclcppdevg_000004.html)推理后端，
-集成了文本检测、分类和识别串联推理。
+### 模型训练与评估-快速指南
 
-具体说明请参考[MindOCR 310推理](docs/cn/inference/inference_tutorial_cn.md)。
+使用`tools/t.py`脚本可以很容易地训练OCR模型，该脚本可支持文本检测和识别模型训练。
```
Comment (wtomin) [inline README_CN.md:101]: 是不是这里把`tools/train.py`误写成`tools/t.py`了

Comment (SamitHuang) [reply README_CN.md]: thanks. fix

File: README_CN.md (hunk 1 above)
Comment (SamitHuang) [reply README_CN.md]: fix

**Pull request merged**